contracts/contracts/strategies/NativeStaking/NativeStakingSSVStrategy.sol

```diff
@@ -113,6 +114,14 @@ contract NativeStakingSSVStrategy is
             _assets,
             _pTokens
         );
+
+        // Approves the SSV Network contract to transfer SSV tokens for deposits
+        IERC20(SSV_TOKEN).approve(SSV_NETWORK, type(uint256).max);
+
+        // Set the FeeAccumulator as the address for SSV validators to send MEV rewards to
+        ISSVNetwork(SSV_NETWORK).setFeeRecipientAddress(
+            FEE_ACCUMULATOR_ADDRESS
+        );
```

Comment:
Should `setFeeRecipient` be made `public` and used here to avoid duplicate code?

Reply:
I thought about adding an internal `_setFeeRecipient` function that was called by `initialize` and `setFeeRecipient` but that seems overkill for one line. You're suggestion of a public `setFeeRecipient` is a better approach to the internal function. But I still think it's overkill for a single line. I think it makes it clearer what `initialize` does.